Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -105,6 +105,7 @@ class IntSize;
 class LayoutPoint;
 class LayoutSize;
 class NotificationResources;
+enum class PlatformEventModifier : uint8_t;
```

Comment:
Move down near other `enum class`.

Reply:
I moved it, but moved it to the wrong place 🤦‍♂️ Will fix shortly

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/e52dff05965acfedc1bef72857dda3d34a9aef8a

Reply:
Scratch that... we don't need this forward decl at all. Removed in https://github.com/WebKit/WebKit/commit/cef350bd53f4108030d9c75e78b458f4bd5f8546